app.yaml

```diff
@@ -1,7 +1,8 @@
 runtime: python37
+instance_class: F4
 automatic_scaling:
   min_instances: 1
-  max_instances: 100
+  max_instances: 20
```

Comment:
How was 20 chosen to be the max_instances?

Reply:
Arbitrary decision by me. I wanted to put some bound on it to cap our potential rate of spend. For calhoun, which uses F1 instances, I chose a crazy-high 100, but I think that service also has a higher chance of high concurrency (workshops, for example). Because F4 instances cost 4x as much as F1, I thought I'd bump it down a little here. Fortunately, this is a pretty quick thing to change if needed.